assets/javascripts/discourse/controllers/preferences-chat.js

```diff
@@ -22,7 +23,7 @@ export default Controller.extend({
 
   @action
   onChangeChatSound(sound) {
-    if (sound) {
+    if (sound && !isTesting()) {
```

Comment:
Just curious, why do we need to disable this?

Reply:
Because when we change the select kit option for the sound this triggers, and then we get an error in the test about the user not having interacted with the document. Pointless to try and play sounds in the test anyway